src/components/Intro.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable jsx-a11y/no-noninteractive-tabindex */
+import React from 'react';
+
+import styled from 'styled-components/macro';
+import { OuterWrapperSection, InnerWrapperSection } from 'styles/ProjectStyles';
+
+const Intro = () => {
+  return (
+    <OuterWrapperSection grey>
+      <InnerWrapperSection>
+        <IntroStyle tabIndex="0">
+          <SpanLine /><SpanDot />
+          Hello! I&apos;m a frontend developer with a background in interior and
```

Comment:
Wow, you got a blank space between the dot and Hello! - i did not manage that. Nice!